controls/3.7/def.cf

```diff
@@ -236,8 +236,9 @@ bundle common def
       # Internal CFEngine log files rotation
       "cfengine_internal_rotate_logs" expression => "any";
 
-      # Enable agent email output (also see mailto, mailfrom and smtpserver)
+      # Enable or disable agent email output (also see mailto, mailfrom and smtpserver)
       "cfengine_internal_agent_email" expression => "any";
+      "cfengine_internal_disable_agent_email" expression => "!any";
```

Comment:
Ya, but if we have a class guard that is defaulted on, its not easy to override that in something like def.json. If things are guarded by enable.!disable we can turn things on by default if we want, and users can still easily disable them with an override class.